examples/mariadb-ephemeral-template.json

```diff
@@ -116,15 +116,15 @@
                   "initialDelaySeconds": 5,
                   "exec": {
                     "command": [ "/bin/sh", "-i", "-c",
-                      "MYSQL_PWD=\"$MYSQL_PASSWORD\" mysqladmin -u $MYSQL_USER ping"]
+                      "MYSQL_PWD=\"$MYSQL_PASSWORD\" && mysqladmin -u $MYSQL_USER ping"]
```

Comment:
I am not an expert in mariadb, but what about to use `export MYSQL_PWD=\"$MYSQL_PASSWORD\" mysqladmin -u $MYSQL_USER ping`

Reply:
Hey @phracek it has been a while since I originally raised this issue so I don't remember all of the details but the problem is that these are two commands without any delineator between them. 

The export changes the scope of the variable but it is still two commands that need some sort of delineator. 

I believe doing a command instead of && would also resolve the issue.